code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1738,6 +1749,17 @@ All effects don't start immediately, but rather get worse over time; the rate is
 		to_chat(L,"<span class='notice'>You notice [mighty_shield] looks worn again. Weird.</span>")
 	..()
 
+/datum/reagent/consumable/ethanol/amaretto_alexander
+	name = "Amaretto Alexander"
+	description = "A weaker version of the Alexander, what it lacks in strength it makes up for in flavor."
+	color = "#DBD5AE"
+	boozepwr = 35
+	quality = DRINK_FANTASTIC
```

Comment:
This is for drinks that either take considerable effort to make or use rare ingredients, these should be DRINK_GOOD at most

Reply:
Haha, what a very clear accident.

Reply:
I set alexander to verygood as its basically just meant to be a weak version of the traditional alexander with it having a nice mood boost instead of the shield percentage change. You cool with that or still want me to bop it back?